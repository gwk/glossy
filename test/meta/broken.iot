{
  'cmd': '$PROJ/iotest.py -no-times test-meta/broken.iot',
  'links' : {'test-meta'},
  'code': 1,
}
