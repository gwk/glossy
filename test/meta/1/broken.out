WARNING: broken test case: test-meta/1/broken
  exception: IotParseError: malformed .iot file: 'test-meta/1/broken.iot'.

################################################################
TESTS FOUND: 1; BROKEN: 1; SKIPPED: 0; FAILED: 0.
